lfucg-exactions/client/sass/_navbar.scss

```diff
@@ -12,10 +12,7 @@ header {
         height: 70px;
     }
     .navbar-brand {
-        background-image: url('../../static/Lexington_Web_Color_Horse_Only.png');
-        background-repeat: no-repeat;
-        background-size: 100%; 
-        width: 50px;
+        /*width: 50px;*/
```

Comment:
related to previous comment, it looks like this used to be a scss solution.  Was this causing problems?